### PR TITLE
Add `follow_imports` option.

### PR DESCRIPTION
**Pull request by edelvalle**
So right now in the go to functionality, if you have something like.

```python
from blabla import Xxx

x = Xxx()
```

And you are in the last like and you try "go to definition" over `Xxx` this thing just goes to the `import` on the top of the file, and does not follow that import to the  actual definition of `Xxx`; and I find that kind of annoying because takes me out of my context, does not go to `Xxx` and I have to hit "go to definition" again, and pray that the module `blabla` does not have `Xxx` as an import also.

So in the past jedi use  to work with `follow_imports=True` by default, and a few months ago it changed. So, this PR makes the default behavior "go the the actual definition" and allows the user to configure to not follow the imports.

### CODE REVIEW

File: sublime_jedi.sublime-settings [hunk 1]
```diff
@@ -11,6 +11,13 @@
     // Additional python package paths.
     "python_package_paths": [],
 
+    // When going to a definition
+    // true: Do you want this to go to where things are defined directly?
```
Comment (srusskih) [inline sublime_jedi.sublime-settings]: Could you please rephrase the help text? 
Something like:
```
"follow_import": true, // go to definition command will follow to the place where it was defined
"follow_import": false, // go to definition command will follow to the place where it was imported
``` 
PS: i'm not the best writer too :)

**Comment by srusskih** [comment]
Could you please add information to README?

**Comment by edelvalle** [comment]
@srusskih  Did the updates you asked for, sorry for the delay ;)

File: sublime_jedi/daemon.py [hunk 2]
```diff
@@ -99,7 +99,7 @@ class JediFacade:
     """
     def __init__(
             self, env, complete_funcargs, source, line, column, filename='',
-            encoding='utf-8', sys_path=None):
+            encoding='utf-8', sys_path=None, follow_imports=False):
```
Comment (srusskih) [inline sublime_jedi/daemon.py]: maybe we can pass this dynamically as parameter to `go_to` function ?

**Comment by edelvalle** [comment]
The commit "Allow passing parameters to daemon requests" opens the door to send a lot of configuration directly to request and not pre-load it with the daemon

**Comment by srusskih** [comment]
@edelvalle  thanks a lot!

**Pull request merged**